src/dosarray_graphing_focus.sh

```diff
@@ -23,8 +36,8 @@ SUFFIX="\n\
 set yrange [0:100] \n\
 set style data lines \n\
 set title '${TITLE}' \n\
```

Comment:
Where does `${TITLE}` get its value -- it was previously `$2`.

Reply:
was earlier a command-line argument to this script. Do we need to take title as an input even though line 40 already sets it to a default value?

Reply:
I see, so line 40 overrides this line? If that's the case, then this line is indeed redundant.
There's a related script that expects to get its title text as a parameter; this script might be inheriting from it.
Let's lean towards flexibility, and provide the title as a parameter.